teku/src/test/java/tech/pegasys/teku/cli/BeaconNodeCommandTest.java

```diff
@@ -332,6 +333,32 @@ public void shouldSetNatMethod() {
         .isEqualTo(NatMethod.UPNP);
   }
 
+  @Test
+  public void doppelgangerDetectionShouldBeDisabledByDefault() {
```

Comment:
It looks like this testcase should be fixed to follow its name goal

Reply:
I fixed it and added another one to test the feature enabling/disabling separately